src/registration/registration.controller.ts

```diff
@@ -1,8 +1,27 @@
-import { Controller } from '@nestjs/common'
+import { Body, Controller, Post, UseGuards } from '@nestjs/common'
+import { ApiTags, ApiBody, ApiResponse } from '@nestjs/swagger'
+import { VolRegisterDto, VolRegisterResponse } from './dto/volregister.dto'
+import { RegistrationService } from './registration.service'
+import { OrgRegisterDto, OrgRegisterResponse } from './dto/orgregister.dto'
 
-@Controller()
+@ApiTags('registration')
+@Controller('registration')
 export class RegistrationController {
-  // TODO Register Org - application
-  // TODO Register Volunteer - application
+  constructor(private readonly registrationService: RegistrationService) {}
+
+  @Post('volregister')
```

Comment:
API: `volregister` -> volunteer
So it will be `api/registration/volunteer`
Same for Organization